habitat-lab/habitat/tasks/rearrange/actions/grip_actions.py

```diff
@@ -100,8 +100,14 @@ def _grasp(self):
         match_coll = None
         contacts = self._sim.get_physics_contact_points()
 
-        robot_id = self._sim.robot.sim_obj.object_id
-        all_gripper_links = list(self._sim.robot.params.gripper_joints)
+        # TODO: the two arguments below should be part of args
+        ee_index = 0
+        index_grasp_manager = 0
```

Comment:
what is the index_grasp_manager?

Reply:
I think we can make it be the same as `ee_index`. Every agent that has multiple arms should have multiple ee_indices and for end effector a grasp_manager, I think these two should be the same